packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component} from 'react'
```

Comment:
let's move this file into the `/sandbox/theatres/components/audio_player/players_by_type` directory